src/App.jsx

```diff
@@ -6,12 +6,21 @@ function App() {
     return (
         <div>
             <div>
-                <button onClick={() => setLayout(1)}> Sphere </button>
                 <button onClick={() => setLayout(0)}> KK </button>
+                <button onClick={() => setLayout(1)}> Sphere </button>
+                <button onClick={() => setLayout(2)}> Random </button>
+                <button onClick={() => setLayout(3)}> Grid </button>
+                <button onClick={() => setLayout(4)}> FR </button>
+                <button onClick={() => setLayout(5)}> Drl </button>
```

Comment:
For now, it's good, it would be better if we could use a `for loop` to generate those buttons. 
And create configs/constants in a separate file that contains button's information.